content/changes.md

```diff
@@ -8,6 +8,28 @@ summary: 'Learn about new features, fixes, and other improvements'
 
 Learn about notable new features, fixes, and other improvements to the application!
 
+## March 31, 2024  
+
+### Improvements 
+
+- **Added preloaded examples** 
+
+  Add preloaded example for Rust and Go 
+
+- **Directory To Tree: allow async function** 
+
+  Directory To Tree required sync code, which was not great for a file system call. We updated the challenge tests to await any promises returned by the solution, giving candidates more flexibility to choose a better practice. 
+
+- **Update Vue challenges to use `<script setup>`** 
+
+  In order to ensure our Vue debugging challenges use modern practices we upgraded all Vue content to use script setup. 
```

Comment:
Technically it's a content improvement that we normally wouldn't report, but in this case I think it's a significant enough language-related upgrade that one of our important customers like IronHack asked about and others probably care about.

That said, it's a bit outdated--I think we added support for Vue 3 a couple years ago and [`<script setup>` in August '23](https://andela.atlassian.net/browse/QUAL-100). This ticket was just to upgrade a few additional challenges' boilerplate to use the syntax. Ideally, we'd report the support for Vue 3 in the backdated August '23 entry.

If you want to keep it here, that's fine, but I'd use:

```suggestion
  We upgraded all of our Vue 3 library content to use `<script setup>`.
```

But better yet, remove it completely in favor of a single Vue 3 `<script setup>` release backdated to August 2023. That's the really key change.